features/step_definitions/your_details_steps.rb

```diff
@@ -64,130 +30,147 @@
-  your_details_page.minimum_contribution_button.set(true)
+When(/^I click the Next button$/) do
+  step 'I submit my details'
 end
 
-When(/^I press next$/) do
-  your_details_page.next_button.click
+When(/^I proceed to the next step$/) do
+  step 'I submit my details'
+end
+
+When(/^I press next and move to your contributions step$/) do
+  step 'I submit my details'
 end
 
 Then(/^I should be able to proceed to the next page$/) do
   expect(page.current_url).to have_content('/your_contributions/new')
 end
 
+Then(/^I should see "([^"]*)" summarised$/) do |my_details|
+  expect(page).to have_content(my_details)
+end
+
 Then(/^I should not be able to choose to make minimum employer contributions$/) do
   expect(your_details_page.minimum_contribution_button).to be_disabled
 end
 
-Then(/^the Step 2 intro paragraph should display my eligible salary$/) do
-  expect(your_contributions_page.contributions_description).to have_content(29124)
+Then(/^the Your Contributions step should tell me my qualifying earnings$/) do
+  expect(your_contributions_page.contributions_description).to have_content('£29,124')
 end
 
-Then(/^the Step 2 intro paragraph should calc my eligible salary$/) do
-  expect(your_contributions_page.contributions_description).to have_content(39124)
+Then(/^I should see that my qualifying earnings is the limit of "([^"]*)"$/) do |limit|
+  expect(your_contributions_page.contributions_description).to have_content(limit)
 end
 
-Then(/^the employee_percent input intro paragraph should display the correct percentage$/) do
-  within('.contributions__source--employee') do
-    expect(your_contributions_page).to have_content('The legal minimum is 1%')
-  end
+Then(/^the Your Contributions step should tell me my qualifying earnings are my salary$/) do
+  expect(your_contributions_page.contributions_description).to have_content("35,000")
 end
 
-Then(/^the employer_percent input intro paragraph should display the correct percentage$/) do
-  within('.contributions__source--employer') do
-    expect(your_contributions_page).to have_content('The legal minimum is 1%')
-  end
+Then(/^I should return to the Your Details step$/) do
+  expect(your_details_page.form).to be_visible
 end
 
-Then(/^I should see "([^"]*)" summarised$/) do |my_details|
-  expect(page).to have_content(my_details)
+Then(/^I should see my current details in the form fields$/) do
+  expect(page).to have_css("input[value='35']")
+  expect( find(:css, 'select#your_details_form_gender').value ).to eq('female')
+  expect(page).to have_css("input[value='35000']")
+  expect( find(:css, 'select#your_details_form_salary_frequency').value ).to eq('year')
+  expect(page).to have_css("input[value='minimum']")
 end
 
-Then(/^I should see that the minimum contribution option should be selected by default$/) do
-  expect(your_details_page.minimum_contribution_button).to be_checked
-end
+# Then(/^the employee_percent input intro paragraph should display the correct percentage$/) do
```

Comment:
So these steps I left in as a reminder. The expectation of these look fairly valid ones, but are no longer written into any feature. I will go back through the code to see whether and where we need to have these step definitions.